bootstrap/internal/controller/agentbootstrapconfig_controller.go

```diff
@@ -443,20 +450,27 @@ func (r *AgentBootstrapConfigReconciler) getInfrastructureRefKey(
 	machine *clusterv1.Machine,
 ) (types.NamespacedName, error) {
 	acp := controlplanev1alpha1.AgentControlPlane{}
+	namespace := acp.Namespace
 	err := util.GetTypedOwner(ctx, r.Client, machine, &acp)
 	if err != nil {
-		// Machine is not owned by ACP, check for MD
-		md := clusterv1.MachineDeployment{}
+		// Machine is not owned by ACP, check for MS
+		md := clusterv1.MachineSet{}
```

Comment:
Super minor nit but if we changed this to MachineSet, maybe the var name could match it? 
```suggestion
		ms := clusterv1.MachineSet{}
```